packages/calypso-e2e/src/lib/blocks/block-flows/instagram.ts

```diff
@@ -57,6 +52,11 @@ export class InstagramBlockFlow implements BlockFlow {
 	 */
 	async validateAfterPublish( context: PublishedPostContext ): Promise< void > {
 		// await context.page.pause();
-		await context.page.locator( '.wp-block-embed-instagram' ).waitFor();
+		await context.page
+			.getByRole( 'figure' )
+			.filter( {
+				hasText: 'View this post on Instagram',
+			} )
+			.waitFor();
```

Comment:
The previous selector was matching on two elements (somehow).

Reply:
Possibly due to proxxied content?